reservations/statistic_utils.py

```diff
@@ -1,52 +1,62 @@
+from reservations.choices import CustomerTypeChoice
 from reservations.models import Reservation, ReservationStatistic, ReservationStatisticsReservationUnit
 
 
 def create_or_update_reservation_statistics(reservation_pk: Reservation) -> None:
     reservation = Reservation.objects.get(pk=reservation_pk)
     recurring = getattr(reservation, "recurring_reservation", None)
+    requires_org_info = (
```

Comment:
organisation name is still wanted if the customer type is non-individual but unreg.association

Reply:
Wait, isn't that what this already does? Or did you mean:

```python
requires_org_name = reservation.reservee_type != CustomerTypeChoice.INDIVIDUAL
requires_org_id = reservation.reservee_is_unregistered_association and requires_org_name
```

Reply:
Yes, that.

Reply:
OK, it was missing a "not" before, but I added this so it should be fixed now.